test/e2e-go/cli/goal/expect/goalDryrunRestTest.exp

```diff
@@ -78,7 +78,7 @@ if { [catch {
     puts "Primary Account Balance: $PRIMARY_ACCOUNT_BALANCE"
 
     set TEAL_PROG_FILE "$TEST_ROOT_DIR/trivial.teal"
-    exec echo int 1 > $TEAL_PROG_FILE
+    exec echo -e "#pragma version 2\nint 1" > $TEAL_PROG_FILE
```

Comment:
travis' macos `echo` does not have `-e` option and fails.
`printf "#pragma version 2\nint 1"` seems working